<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectionChooser</class>
 <widget class="QDialog" name="ProjectionChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="chkPredefined">
       <property name="text">
        <string>Predefined</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="chkStandard">
       <property name="text">
        <string>Standard (EPSG:...)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="chkCustom">
       <property name="text">
        <string>Custom PROJ4(+proj=...)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtStandard"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtCustom"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbPredefined">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectionChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectionChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPredefined</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>chkPredefined</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtStandard</sender>
   <signal>textEdited(QString)</signal>
   <receiver>chkStandard</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtCustom</sender>
   <signal>textEdited(QString)</signal>
   <receiver>chkCustom</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
